python/graphstorm/run/launch.py

```diff
@@ -419,6 +419,8 @@ def construct_dgl_server_env_vars(
 
     """
     server_env_vars_template = (
+        #"NVIDIA_VISIBLE_DEVICES={NVIDIA_VISIBLE_DEVICES} " temporary flags for testing
```

Comment:
Should I remove those flags or might their be useful for later eg. for testing?

Reply:
I would suggest to remove them and find other way for testing.